docs/user-guide/nginx-configuration/configmap.md

```diff
@@ -316,6 +317,12 @@ Disable listening on IPV6. _**default:**_ `false`; IPv6 listening is enabled
 
 Disable IPV6 for nginx DNS resolver. _**default:**_ `false`; IPv6 resolving enabled.
 
+## enable-snippet-directives
+
+Enables Ingress to parse and add *-snippet annotations/directives created by the user. _**default:**_ `true`;
+Obs.: We recommend enabling this option only if you TRUST users with permission to create Ingress objects, as this 
```

Comment:
What is `Obs.`? Maybe not shorten it? For the text I'd probably go with:

```
Allows snippets annotations that can inject any NGINX configuration. Set this to `false` if you do not trust the users configuring the ingresses. _**default:**_ `true`;
```

Reply:
yeah, agreed. I will change this one here.

Obs is a shorten for "Observação" in portuguese and my head though this made all sense in english...hahaha

I will change here for WARNING

@strongjz @cpanato `enable-snippet-directives` vs `allow-snippet-annotations`

I'm tempted now in changing to `allow-snippet-annotations` as Elvin proposed.

Reply:
> Observação

👍🏼 i learned some portuguese